assembl/static2/css/components/posts.scss

```diff
@@ -179,13 +183,15 @@ $deleted-post-border: 1px solid $grey1 !default;
         .label {
             color: $grey2;
             font-size: $font-size-xs;
+            font-family: 'LatoLight'
         }
 
         .badges {
             padding: 10px 0;
 
             .badge {
-                background-color: $grey1;
+              display: inline;
+                background-color: #e5e5e5;
```

Comment:
We try to keep to a colour palette that is defined the `variables.scss`. We have 6 grey variables. If this colour is to be used elsewhere, then it should warrant the creation of another grey variable. If not, then find the closest grey variable that already exists to this value. Thanks